File: 기초-브론즈 문제/백준_4344_평균은 넘겠지/평균은 넘겠지.java
import java.util.*;

public class Main{

    public static void main(String[] args){
        Scanner s = new Scanner(System.in);
        int m = s.nextInt();
        while(m-- > 0){
            int n = s.nextInt();
            int[] arr = new int[n];
            int ever = 0;
            for(int i = 0; i < n; i++) {
                arr[i] = s.nextInt();
                ever += arr[i];
            }
            ever /= n;
            int cnt = 0;
            for(int i = 0; i < arr.length; i++)
                if(arr[i] > ever) cnt++;
            System.out.format("%.3f%%\n", (float)cnt*100/n);
        }
    }
}

File: 기초-브론즈 문제/백준_4344_평균은 넘겠지/평균은 넘겠지.cpp
#include <iostream>

using namespace std;


int main() {
    int t = 0;
    scanf("%d", &t);

    int stu_score[1001] = {0};

    int n = 0;
    float ever = 0;
    while(t--){
        scanf("%d", &n);
        int sum = 0;
        for(int i =0; i<n; i++) {
            scanf("%d", &stu_score[i]);
            sum += stu_score[i];
            ever = (float)sum/n;
        }
        int cnt = 0;
        for(int i =0; i< n; i++){
            if(stu_score[i] > ever) cnt++;
        }

        printf("%.3f%%\n", ((float)cnt*100/n));
    }




}

